code/__HELPERS/icons.dm

```diff
@@ -813,11 +813,15 @@ The _flatIcons list is a cache for generated icon files.
 			if(4)	I.pixel_y++
 		overlays += I//And finally add the overlay.
 
-/proc/getHologramIcon(icon/A, safety=1)//If safety is on, a new icon is not created.
+/proc/getHologramIcon(icon/A, double, safety=1)//If safety is on, a new icon is not created. Double is for 32x64 icons.

```

Comment:
you should really probably just make a new arg to specify what file it uses for alpha_mask instead of hardcoding 32x64 icons in